.github/workflows/d.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 
       # NOTE use KyleMayes/install-llvm-action once https://github.com/KyleMayes/install-llvm-action/pull/11 is merged
       - name: Setup LLVM
-        uses: aminya/install-llvm-action@defeault-directory
```

Comment:
If this was mispelled, did it work before? Is it even needed?

Reply:
This is needed. The typo was actually in the branch name. I made a new branch with the correct name.